src/Native/MatrixFactorizationNative/UnmanagedMemory.cpp

```diff
@@ -9,25 +9,48 @@
 
 using namespace mf;
 
+EXPORT_API(mf_parameter) make_param(const mf_parameter_bridge *param_bridge)
```

Comment:
I don't think this needs `EXPORT_API` does it?

Reply:
Yes!